### PR TITLE
Noisy Networks for Exploration

### PR DESCRIPTION
**Pull request by ghost**
Hello, 

  I implemented NoisyNet layers into Keras API. It's useful for users, that experiments with reinforcement learning algorithms (NoisyDQN or NoisyA3C). For more information, I made a description of those layers in the code.

## Links
[Noisy Networks for Exploration](https://arxiv.org/abs/1706.10295)

### CODE REVIEW

**Comment by markub3327** [comment]
Hello, 

&emsp; I tested it on my Bachelor's project and the results are here. I found here small mistake with `self.kernel_epsilon.assign(w_eps)` and `self.bias_epsilon.assign(b_eps)` -- bad spaces before this, which cause problems...

## Chart
loss
score


## Code
[Noisy DQN MazeSolver](https://github.com/markub3327/DQN_MazeSolver)

**Comment by markub3327** [comment]
Hello,
   
> what do you think about this new feature?

**Comment by pavithrasv** [comment]
I think this should be added to the Keras layers in TensorFlow add-ons repository.

**Comment by markub3327** [comment]
Where I found the TensorFlow add-ons repository, please?

**Comment by markub3327** [comment]
I made new PR here: https://github.com/tensorflow/addons/pull/2202

I think that this PR should be closed ... Any ideas?
